temporalio/contrib/openai_agents/_temporal_openai_agents.py

```diff
@@ -281,9 +315,22 @@ def configure_worker(self, config: WorkerConfig) -> WorkerConfig:
         config["interceptors"] = list(config.get("interceptors") or []) + [
             OpenAIAgentsTracingInterceptor()
         ]
-        config["activities"] = list(config.get("activities") or []) + [
-            ModelActivity(self._model_provider).invoke_model_activity
-        ]
+        new_activities = [ModelActivity(self._model_provider).invoke_model_activity]
+        for mcp_server in self._mcp_servers:
```

Comment:
I'd go one further and say these _must_ be one of the Temporal instances (of now, we can weaken later if we want) and also to confirm there are no duplicate names (and/or provide a way to override the name just for our lookup).